packages/components/radio-group/src/utils/helper-functions.js

```diff
@@ -0,0 +1,12 @@
+import { keyboardKeys } from './constants';
+
+export default function getDirectionByKey(keyCode) {
+	switch (keyCode) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "default" clause to this "switch" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ASwitchWithoutDefault)